com.gitlab.coringao.cavestory-nx.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.gitlab.coringao.cavestory-nx",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cavestory-nx",
+  "finish-args": [
+    "--device=all",
+    "--socket=pulseaudio",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "cavestory-nx",
+      "no-make-install": true,
+      "buildsystem": "cmake-ninja",
+      "build-commands": [
+        "install -D cavestory-nx /app/bin/cavestory-nx",
+        "cp -R data /app/bin/data",
+        "install -Dm644 cavestory-nx.desktop /app/share/applications/com.gitlab.coringao.cavestory-nx.desktop",
+        "install -Dm644 data/cavestory-nx.png /app/share/icons/hicolor/512x512/apps/com.gitlab.coringao.cavestory-nx.png",
```

Comment:
You can provide smaller icons using `rsvg-convert`.

See also:
https://github.com/scx/flathub/blob/14143a6edd6c83559455f97436101670e13e1345/Aegisub.yaml#L66-L72

Reply:
Can `rsvg-convert` resize a png?

Reply:
Sorry, I thought SVG was there.